src/style.css

```
@@ -1,3 +1,73 @@
-body {
+body,
+html {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  --link-сolor: #000;
+  --link-сolor-active: #00acdc;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  margin: auto;
+  flex-shrink: 0;
+  font-size: 0;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+.logo {
+  margin: 10px auto 10px 50px;
+  align-self: center;
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+}
+
+.nav {
+  margin-right: 40px;
+  flex-shrink: 0;
+}
+
+.nav-menu {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__list {
+  position: relative;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+  padding: 0 10px;
+  margin: 0 10px;
+}
+
+a.nav__list {
+  display: inline-block;
+  text-decoration: none;
+  color: var(--link-сolor);
+  padding: 23px 0;
+}
```

Do you have another `nav__list` class? Don't use tags for styling 
```suggestion
nav__list {
  display: inline-block;
  text-decoration: none;
  color: var(--link-сolor);
  padding: 23px 0;
}
```